<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>veritas_gui</class>
 <widget class="QWidget" name="veritas_gui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>316</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Veritas</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_Start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select your files: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_Browse">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_Help">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
